### PR TITLE
fix: build pipe

### PR DESCRIPTION
**Pull request by Zaista**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,14 +11,14 @@ jobs:
...
+        cypress-version: [10.11.0, 11.2.0, 12.17.4, 13.2.0]
 
     steps:
       - uses: actions/checkout@v2
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-          # - run: bash test.sh ${{ matrix.cypress-version }}
+      - run: npm install cypress@${{ matrix.cypress-version }}
```
Comment (area73) [inline .github/workflows/ci.yml:22]: 👌

Comment (area73) [reply .github/workflows/ci.yml:22]: You can safely remove line 23

You don't need to do again a "npm install"

**Comment by area73** [comment]
Please, don't forget to remove dockerfile and test.sh

**Comment by Zaista** [comment]
@area73 Any ideas why the ci build is failing? If I remove `@ts-expect-error` comment locally, I can't get the build to work. I don't really get what's happening

**Pull request merged**